sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -69,6 +69,16 @@ public partial class DetectLanguageResultCollection : System.Collections.ObjectM
         public string ModelVersion { get { throw null; } }
         public Azure.AI.TextAnalytics.TextDocumentBatchStatistics Statistics { get { throw null; } }
     }
+    public partial class DocumentHealthcareResult
+    {
+        public DocumentHealthcareResult(string id, Azure.AI.TextAnalytics.TextAnalyticsError textAnalyticsError) { }
```

Comment:
I missed this one. Could you make the constructor internal? Because this is an output type, we don't want users to be able to instantiate this class